teacher_class.rb

```diff
@@ -0,0 +1,18 @@
+require_relative 'person_class'
+
+class Teacher < Person
+  attr_accessor :specialization
+
+  def initialize(specialization)
+    @specialization = specialization
+    super(
+      age,
+      name,
+      parent_permission
+    )
```

Comment:
- [OPTIONAL] Same here, let's add the age, name, and parent_permission as parameters of the teacher Constructor. It might look like this:

```ruby
  def initialize(age, specialization, name = "Unknown", parent_permission = true)
    super(age, name, parent_permission)
    @specialization = specialization
  end
```